### DISCUSSION TITLE
Aug 23 show and tell notes

### DISCUSSION THREAD

**Discussion by isabela-pf** [Quansight/Quansight.github.io#42] [Show and tell]
Aug 23 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

Please do not disclose confidential customer information.

**Comment by isabela-pf** [comment]
First ten minutes: fun banter! Emojis, ASCII art, and more, oh my!

**Reply by tonyfast** [reply]
- https://gitmoji.dev/
- https://www.amazon.com/Because-Internet-Understanding-Rules-Language/dp/0735210934
- https://github.com/honno/cool-s
- https://github.com/willmcgugan/rich/pull/1419

**Comment by tonyfast** [comment]
- https://pypi.org/project/bitarray/
- https://docs.python.org/3/library/mmap.html
- https://docs.python.org/3.1/library/functions.html#bytearray
- https://docs.python.org/3.1/library/functions.html#memoryview
